src/Dunet.csproj

```
@@ -33,6 +34,7 @@
         <SignAssembly>False</SignAssembly>
         <Version>1.0.0</Version>
         <NeutralLanguage>en</NeutralLanguage>
+        <DevelopmentDependency>true</DevelopmentDependency>
     </PropertyGroup>
```

Getting the NU5128 warning in the build step. Here's some more info on it: https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128

I think we can safely suppress this warning with the following:

```suggestion        
        <NoWarn>$(NoWarn);NU5128</NoWarn>
    </PropertyGroup>
```

I ran some manual integration tests with the change and everything works great. I'll approve with the above change.

Thanks again for the contribution!

---

Done, thanks :)